ops/ccs-ops-misc/load_test/common/pull_par_adj_hashed_mbis.py

```diff
@@ -0,0 +1,36 @@
+import os
+import psycopg2 as p
+import yaml
+from common import config
+
+
+def loadData():
+    ## Load configuration data, like db creds
+    configFile = config.load()
+
+    ## if we failed to load the config, bail out
+    if configFile is None:
+        return -1
+
+    fileName = "mbis.txt"
+    beneQuery = "select f.\"mbiHash\" from \"pre_adj\".\"FissClaims\" f where f.\"mbi\" IS NOT NULL union select m.\"idrClaimMbiHash\" from \"pre_adj\".\"McsClaims\" m where m.\"idrClaimMbi\" IS NOT NULL;"
+
+    ## Make the query to the DB
+    conn = p.connect(
+        user=configFile["dbUsername"],
+        password=configFile["dbPassword"],
+        host=configFile["dbHost"],
+        port='5432',
+        database='fhirdb'
+    )
```

Comment:
Does this need to be closed or cleaned up in some way after reading the data?

Reply:
I didn't see anything in the other scripts, looking at the docs, it seems to manage it's own connection pool, I didn't see anything about manually cleaning up the connections at a glance